components/button/Button.module.css

```diff
@@ -14,6 +14,15 @@
   transition: var(--bottom-transition);
 }
 
+.btn:disabled {
+  background-image: linear-gradient(
+      var(--placeholder-color),
+      var(--placeholder-color)
+    ),
+    linear-gradient(var(--placeholder-color), var(--placeholder-color));
+  color: var(--placeholder-color);
+}
```

Comment:
I don't understand this. If you have a linear-gradient with the same colors it isn't a gradient at all.

Reply:
Yeah, that's a bit of CSS magic.
Doens't work if it isn't written like this.
See this source: https://stackoverflow.com/questions/4131490/any-way-to-limit-border-length/42157360#42157360